ssh/tailssh/tailssh.go

```diff
@@ -200,18 +202,32 @@ func (srv *server) handleSSH(s ssh.Session) {
 		s.Exit(1)
 		return
 	}
-	if action.Message != "" {
-		io.WriteString(s.Stderr(), strings.Replace(action.Message, "\n", "\r\n", -1))
-	}
-	if action.Reject {
-		logf("ssh: access denied for %q from %v", ci.uprof.LoginName, ci.src.IP())
-		s.Exit(1)
-		return
-	}
-	if !action.Accept || action.HoldAndDelegate != "" {
-		fmt.Fprintf(s, "TODO: other SSHAction outcomes")
-		s.Exit(1)
-		return
+ProcessAction:
```

Comment:
Can this keep waiting forever? looks like it can, add comment and/or way to make it not.

Reply:
fetchSSHAction returns an error on context timeout

Reply:
yes, but on success it can return an Action which points to the same HoldAndDelegate URL. So it'll keep fetching the same thing over and over again

Reply:
Done.